gnomad_qc/v5/sample_qc/create_sample_qc_metadata_ht.py

```diff
@@ -182,6 +174,53 @@ def annotate_genetic_ancestry(
     return meta_ht
 
 
+def update_hgdp_tgp_outlier_annotation(
+    meta_ht: hl.Table, v4_meta_ht: hl.Table
+) -> hl.Table:
+    """
+    Update `outlier_filters` in ht.sample_filters based on the annotations in hgdp_tgp_meta_updated.
+
+    .. note::
+    # Note: For 'gnomad' samples, we cannot rely solely on the length of 'outlier_filters' because some v3-filtered samples were
+    # rescued in v4 and still retain populated 'outlier_filters'. The 'hgdp_tgp_meta_updated' contains an updated 'outlier' bool. For samples
+    present in "hgdp_tgp_meta_updated" and marked as either "hgdp" or "tgp" in the v4 meta subsets, if the updated outlier annotation is True,
+    outlier_filters will be populated with hgdp_tgp_outlier. If the updated outlier annotation is False, the outlier_filters annotation will be
+    updated to an empty set.
```

Comment:
```suggestion
    .. note::
    
        For 'gnomad' samples, we cannot rely solely on the length of 'outlier_filters' because some v3-filtered samples were
        rescued in v4 and still retain populated 'outlier_filters'. The 'hgdp_tgp_meta_updated' contains an updated 'outlier' bool. For samples
    present in "hgdp_tgp_meta_updated" and marked as either "hgdp" or "tgp" in the v4 meta subsets, if the updated outlier annotation is True,
    outlier_filters will be populated with hgdp_tgp_outlier. If the updated outlier annotation is False, the outlier_filters annotation will be
    updated to an empty set.
```